apps/src/calc/expressionNode.js

```diff
@@ -355,8 +359,9 @@ ExpressionNode.prototype.getTokenListDiff = function (other) {
  * @param {boolean} markDeepest Mark tokens in the deepest descendant
  */
 ExpressionNode.prototype.getTokenList = function (markDeepest) {
-  var depth = this.depth();
-  if (depth <= 1) {
+  if (!markDeepest || this.depth() <= 1) {
+    // if markDeepest is true, we diff against null so that everything is marked
```

Comment:
Would this logic read a bit clearer if it were structured similar to how the comment explains it?

```
if (markDeepest) {
  // diff against null so everything is marked
} else {
  // diff against ourself, nothing will be marked
}